client/me/help/main.jsx

```diff
@@ -128,12 +128,17 @@ const Help = React.createClass( {
 
 		return (
 			<HelpTeaserButton
+				onClick={ this.trackCoursesButtonClick }
 				href="/help/courses"
 				title={ this.translate( 'Courses' ) }
 				description={ this.translate( 'Learn how to make the most of your site with these courses and webinars' ) }/>
 		);
 	},
 
+	trackCoursesButtonClick() {
+		analytics.tracks.recordEvent( 'calypso_help_courses_click' );
```

Comment:
Do we want to send the `isBusinessPlan` property with this one too, in case we decide in the future to make these buttons clickable (but leading to say a page with info about the plan) for non-business plan users?